index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter&display=swap"
+      rel="stylesheet"
+    />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="nav-section">
+      <div class="welcome-heading">
+        <h1>Welcome</h1>
```

Comment:
- [ ] Please use an `a` tag for the logo, `h1` tag is not the right semantic tag

Reply:
Are you sure on this because its a greeting message. It can not be a link.

Reply:
Ok I get it now, it has to be a link. Fixing it :)